package.json

```diff
@@ -35,12 +35,12 @@
   },
   "scripts": {
     "postinstall": "electron-builder install-app-deps",
-    "build": "tsc",
+    "build-main": "tsc",
     "watch": "tsc -w",
     "lint": "eslint -c .eslintrc --ext .ts ./src",
-    "start": "concurrently \"yarn build && electron ./app/main.js\" \"cd bada-frame && yarn dev\"",
-    "build-source" : "cd bada-frame && yarn && yarn build && cd ..",
-    "test-release": "yarn && yarn build-source && yarn build && electron-builder",
+    "start": "concurrently \"yarn && yarn run build-main && electron ./app/main/main.js\" \"cd src/renderer && yarn && yarn dev\"",
```

Comment:
why do we need to `yarn run `here? 
also, it would be nice if you add a pre-build script and run `yarn`  and `cd src/renderer && yarn` in it so you don't have to repeat it multiple times